modules/web/src/app/core/components/sidenav/template.html

```diff
@@ -235,20 +228,21 @@
          target="_blank"
          rel="noopener">
         <i [ngStyle]="getCustomLinkIconStyle(link)"></i>
-        <span *ngIf="!isSidenavCollapsed()">{{link.label}}</span>
+        <span *ngIf="!isSidenavCollapsed">{{link.label}}</span>
       </a>
     </div>
   </ng-container>
 
   <div fxFlex></div>
 
   <div [matTooltip]="getTooltip(view.Projects)"
-       class="menu-item">
+       class="go-back"
+       [ngClass]="isSidenavCollapsed && 'go-back-collapsed'">
```

Comment:
Have you verified if this class is being applied this way?

Reply:
yes it's working fine